### PR TITLE
Add commercial FAQ

### PR DESCRIPTION
**Pull request by mbj**

### CODE REVIEW

File: docs/commercial.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+Will not be forced to get a mutant subscription.
+
+What happens if my subsription lapses?
+--------------------------------------
+
+If your credit card cannot be charged, even after retries: The subscription will be
+considered inactive and you loose access to mutants license gem. Which prevents `bundle install`
+and adjacent commands to succeed.
+
+**Please note that pricing can chance**. Once you purchase a subscription, you get hat price
```
Comment (dgollahon) [inline docs/commercial.md:60]: FYI i think `hat` is meant to be `that`

File: docs/commercial.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+forever **as long you pay on time**. If your subscription expires for a non-payment, you will lose
+that pricing and need to repurchase at current pricing.
+
+What does the license require me to do?
+---------------------------------------
+
+Your purchase gets you unique access credentials for accessing hte license gem which enables
```
Comment (dgollahon) [inline docs/commercial.md:67]: `hte` -> `the`

File: docs/commercial.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+server that enables mutants functionality in commercial repositories.
+
+
+How do I determine the number of required subscriptions?
+--------------------------------------------------------
+
+Collect the git author email from each of your developers that need to use utant.
+Use `git config --get user.email`.  Count that list.
+
+Mutant on CI will fork on any commit authored by a developer in that list.
```
Comment (vaporyhumo) [inline docs/commercial.md:48]: Maybe you meant "will **work**"?

File: docs/commercial.md (hunk 1 above)
Comment (mbj) [reply docs/commercial.md:60]: thanks.

File: docs/commercial.md (hunk 2 above)
Comment (mbj) [reply docs/commercial.md:67]: thanks.

File: docs/commercial.md (hunk 3 above)
Comment (mbj) [reply docs/commercial.md:48]: thanks.

**Comment by mbj** [comment]
@dgollahon, @vaporyhumo thanks for the typo flags, fixed in: https://github.com/mbj/mutant/pull/1266

**Pull request merged**